docs/3_guides/0_submitting_proofs.md

```diff
@@ -188,11 +192,11 @@ aligned submit \
 --proof ./scripts/test_files/risc_zero/fibonacci_proof_generator/risc_zero_fibonacci.proof \
 --vm_program ./scripts/test_files/risc_zero/fibonacci_proof_generator/fibonacci_id.bin \
 --public_input ./scripts/test_files/risc_zero/fibonacci_proof_generator/risc_zero_fibonacci.pub \
---batcher_url wss://batcher.alignedlayer.com \
+--batcher_url wss://mainnet.batcher.alignedlayer.com \
```

Comment:
Use holesky on the predefined examples. It could lead to users sending funds in mainnet by error